### PR TITLE
WEB-739 fix aspect ratio wrapper

### PR DESCRIPTION
**Pull request by sergivillar**

### CODE REVIEW

File: src/utils/aspect-ratio-support.tsx [hunk 1]
```diff
@@ -74,8 +74,8 @@ export const AspectRatioElement = (props: AspectRatioElementProps): JSX.Element
             style: {
                 ...(needsWrapper
                     ? {
-                          position: aspectRatio ? 'absolute' : 'static',
                           ...props.style,
+                          position: aspectRatio ? 'absolute' : 'static',
```
[the remark this answers is not in the document]
Comment (sergivillar) [reply src/utils/aspect-ratio-support.tsx:78]: this only applies to the wrapper when aspect ratio is used
is aspect ratio is supported, this changes wont apply

and yes, I talked with Victor and this fix the issue in devices that does not support aspect ratio

[the remark this answers is not in the document]
Comment (atabel) [reply src/utils/aspect-ratio-support.tsx:78]: hmmm... but does Skeleton/ImageError still work in images with aspect ratio?

[the remark this answers is not in the document]
Comment (sergivillar) [reply src/utils/aspect-ratio-support.tsx:78]: yep, booth keeps working (ImageError has a screenshot test too)

**Review by atabel — APPROVED** [review]

**Pull request merged**